relays/ethereum/Cargo.toml

```diff
@@ -11,11 +11,17 @@ async-stream = "0.2.0"
 clap = { version = "2.33.0", features = ["yaml"] }
 codec = { package = "parity-scale-codec", version = "1.0.0" }
 env_logger = "0.7.0"
+ethabi = "12.0"
+ethabi-contract = "11.0"
+ethabi-derive = "11.0"
+ethereum-tx-sign = { git = "https://github.com/svyatonik/ethereum-tx-sign.git", branch = "up-ethereum-types" }
```

Comment:
The biggest change of latest merge-with-master - PR is already opened in original repo, but let's refer to my branch for now